libd-llvm/src/d/llvm/expression.d

```diff
@@ -468,6 +468,7 @@ struct ExpressionGen {
 			auto thisPtr = LLVMBuildExtractValue(builder, dg, m.hasContext, "");
 			auto vtblPtr = LLVMBuildStructGEP(builder, thisPtr, 0, "");
 			auto vtbl = LLVMBuildLoad(builder, vtblPtr, "vtbl");
```

Comment:
The invariant.group is not guaranteed to be 16 .
Plus you neec to load the "fun"-pointer with the same invariant.group.